dask/array/creation.py

```diff
@@ -379,17 +379,26 @@ def indices(dimensions, dtype=int, chunks='auto'):
         The shape of the index grid.
     dtype : dtype, optional
         Type to use for the array. Default is ``int``.
-    chunks : sequence of ints
-        The number of samples on each block. Note that the last block will have
-        fewer samples if ``len(array) % chunks != 0``.
+    chunks : sequence of ints, str
+        The size of each block.  Must be one of the following forms:
+
+        -   A blocksize like (500, 1000)
+        -   A size in bytes, like "100 MiB" which will choose a uniform
+            block-like shape
+        -   The word "auto" which acts like the above, but uses a configuration
+            value ``array.chunk-size`` for the chunk size
+        Note that the last block will have fewer samples if ``len(array) % chunks != 0``.
 
     Returns
     -------
     grid : dask array
     """
     dimensions = tuple(dimensions)
     dtype = np.dtype(dtype)
-    chunks = tuple(chunks)
+    if isinstance(chunks, str):
+        chunks = normalize_chunks(chunks, shape=dimensions, dtype=dtype)
+    else:
+        chunks = tuple(chunks)
```

Comment:
Ideally this would be ...


```python
chunks = normalize_chunks(chunks, shape=dimensinos, dtype=dtype)
```

We shouldn't have to have a condition around calling a `normalize` function.  It should ideally be safe in either event.

Reply:
Good point, thanks for the feedback. I've updated accordingly